src/web/components/App.tsx

```diff
@@ -317,7 +317,7 @@ export const App = ({ CAPI, NAV }: Props) => {
                 </Lazy>
             </Portal>
             <Portal root="cmp">
-                <CMP cmpUi={CAPI.config.cmpUi} />
+                <>{CAPI.config.cmpUi && <CMP />}</>
```

Comment:
We could also wrap the `<Portal>` in a check for `cmpUI`

Reply:
good point 👍